core/src/main/java/com/yahoo/oak/InternalOakHashMap.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright 2020, Verizon Media.
+ * Licensed under the terms of the Apache 2.0 license.
+ * Please see LICENSE file in the project root for terms.
+ */
+
+package com.yahoo.oak;
+
+import java.io.Closeable;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.function.BiConsumer;
+import java.util.function.BiFunction;
+import java.util.function.Consumer;
+import java.util.function.Function;
+
+public class InternalOakHashMap<K , V> implements Closeable {
```

Comment:
As `internalOakMap` was temporarily added to `OakHashMap` instead of `internalOakHashMap` this unreachable class should be better removed. It will just cause test coverage check to fail. We will add it together with the implementation.

Reply:
done